docs/INSTALL.md

```diff
@@ -90,10 +90,11 @@ Installation instructions can be found [here](https://flatpak.org/setup/)
 
 The following Flatpak SDKs are required:
 ```
-flatpak install org.freedesktop.Platform//20.08
-flatpak install org.freedesktop.Sdk//20.08
+flatpak install org.kde.Platform//5.15
```

Comment:
Could we keep the old install instructions, and add a note regarding the two versions, and its different pre-requirements?